src/app/services/canvas-shapes-service.service.ts

```diff
@@ -46,10 +45,10 @@ export class CanvasShapesServiceService {
       fill: 'transparent',
       stroke: '#000',
     });
-    this.subject.next({ shape: this.shape, name: 'Circle' });
+    this.subject.next(this.shape);
   }
 
-  drawShapeOnCanvas(): Observable<object> {
+  drawShapeOnCanvas(): Observable<Object> {
```

Comment:
Object is general object, You can use fabric.Object

Reply:
Thanks..Will change that